targeteditor/src/main/scala/explore/targeteditor/SkyPlotSection.scala

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.targeteditor
+
+import java.time.LocalDate
+import java.time.ZoneId
+import java.time.ZoneOffset
+
+import cats.Eq
+import cats.implicits._
+import explore.components.ui.GPPStyles
+import explore.model.reusability._
+import gem.enum.Site
+import gpp.ui.reusability._
+import gsp.math.Coordinates
+import japgolly.scalajs.react.CatsReact._
+import japgolly.scalajs.react.MonocleReact._
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import monocle.macros.Lenses
+import react.common.ReactProps
+import react.semanticui.modules.checkbox.Checkbox
+
+final case class SkyPlotSection(
+  coords: Coordinates
+) extends ReactProps[SkyPlotSection](SkyPlotSection.component)
+
+object SkyPlotSection {
+  type Props = SkyPlotSection
+
+  protected sealed trait TimeDisplay
+  protected object TimeDisplay {
+    final case object UTC  extends TimeDisplay
+    final case object Site extends TimeDisplay
+
+    implicit val TimeDisplayEq: Eq[TimeDisplay]             = Eq.fromUniversalEquals
```

Comment:
I suppose this can be `Eq.byRef` as they are singleton objects, but don't worry just a detail

Reply:
Yes, good catch.

Reply:
Oh right, there should be one :D